src/algorithms/es256.rs

```diff
@@ -26,6 +27,28 @@ impl P256PublicKey {
         Ok(P256PublicKey(p256_pk))
     }
 
+    pub fn from_jwk_str(jwk: &str) -> Result<Self, Error> {
+        let pk = PublicKey::from_jwk_str(jwk).map_err(|_| JWTError::InvalidPublicKey)?;
+        let pk = ecdsa::VerifyingKey::from(pk);
+        Ok(P256PublicKey(pk))
+    }
+
+    pub fn from_jwk(jwk: &JwkEcKey) -> Result<Self, Error> {
+        let pk = PublicKey::from_jwk(jwk).map_err(|_| JWTError::InvalidPublicKey)?;
+        let pk = ecdsa::VerifyingKey::from(pk);
+        Ok(P256PublicKey(pk))
+    }
+
+    pub fn to_jwk(&self) -> JwkEcKey {
+        let pk = PublicKey::<p256::NistP256>::from(&self.0);
+        pk.to_jwk()
+    }
+
+    pub fn to_jwk_str(&self) -> String {
+        let pk = PublicKey::<p256::NistP256>::from(&self.0);
+        pk.to_jwk_string()
+    }
```

Comment:
Hmm, this makes me think I should add a `jwk` feature to the `ecdsa` crate...

Reply:
yeah, that could be great!